lib/attachinary/utils.rb

```diff
@@ -25,7 +25,9 @@ def self.permitted_file_params(hash)
       if Rails::VERSION::MAJOR == 3
         Attachinary::File.new hash.slice(*Attachinary::File.attr_accessible[:default].to_a)
       else
-        ActionController::Parameters.new(hash).permit(
+        # permit_without_raise is a monkeypatch found in reverb/lib/monkeypatches/action_controller.rb
+        # It has the same behavior as ActionController::Parameters#permit except that it does not rais or log ActionController::UnpermittedParameters
+        ActionController::Parameters.new(hash).permit_without_raise(
```

Comment:
would it make sense to define this in core as an attachinary monkey patch?

Reply:
@mcostanza  You mean different than this? https://github.com/reverbdotcom/reverb/pull/22709/files#diff-480f4c97dc2bc2d9ab5930c211281affR39

Reply:
I was thinking move this code to `lib/monkeypatches/attachinary.rb` within core, since it's relying on another monkeypatch method that only exists in core.

Reply:
Ooh, I like that idea actually. It makes it more self-contained and we don't have to touch this gem.  Easier to rollback too.
The only drawback is that then we have 2 monkey patches, but we own this attachinary code at this point anyway.
Because ideally this should be temporary I'm going to go with your approach.

Reply:
yeah either way we need 2 patches, might as well have them next to each other in core.